src/theory/quantifiers/first_order_model.h

```diff
@@ -103,7 +103,7 @@ class FirstOrderModel : public TheoryModel
   /** is model basis term */
   bool isModelBasisTerm(Node n);
   /** get model basis term for op */
-  Node getModelBasisOpTerm(Node op);
+  Node getModelBasisOp(Node op);
```

Comment:
Was this change intentional?

Reply:
It sure wasn't -- sloppy `ag`/`sed` on my part. Thanks for catching that -- should have looked at the changed files. Fixed with d61f876d59aef39563da053b15a42fcc9aee0122 and a10fc0ce7b45befe1a67850156ec38005802fdbe.